### PR TITLE
Update Connector release status

### PR DESCRIPTION
**Pull request by CalvinKirs**
After production testing and feedback from community users, we have updated the health status of some Connectors

### CODE REVIEW

File: docs/en/Connector-v2-release-state.md [hunk 1]
```diff
@@ -11,55 +11,74 @@ SeaTunnel uses a grading system for connectors to help you understand what to ex
...
+| [LocalFile](connector-v2/sink/LocalFile.md)                 | Sink   | GA     | 2.2.0-beta      |
+| [LocalFile](connector-v2/source/LocalFile.md)               | Source | GA     | 2.2.0-beta      |
+| [Maxcompute](connector-v2/source/Maxcompute.md)             | Source | Alpha  | 2.3.0           |
+| [Maxcompute](connector-v2/sink/Maxcompute.md)               | Sink   | Alpha  | 2.3.0           |
 | [MongoDB](connector-v2/source/MongoDB.md)                   | Source | Beta   | 2.2.0-beta      |
 | [MongoDB](connector-v2/sink/MongoDB.md)                     | Sink   | Beta   | 2.2.0-beta      |
 | [MyHours](connector-v2/source/MyHours.md)                   | Source | Alpha  | 2.2.0-beta      |
-| [Neo4j](connector-v2/sink/Neo4j.md)                         | Sink   | Alpha  | 2.2.0-beta      |
-| [OssFile](connector-v2/sink/OssFile.md)                     | Sink   | Alpha  | 2.2.0-beta      |
+| [MySqlCDC](connector-v2/source/MySQL-CDC.md)                | Source | GA     | 2.3.0           |
```
Comment (hailin0) [inline docs/en/Connector-v2-release-state.md:60]: @ashulin please confirm `GA`

File: docs/en/Connector-v2-release-state.md [hunk 2]
```diff
@@ -11,55 +11,74 @@ SeaTunnel uses a grading system for connectors to help you understand what to ex
...
 | Connector Name                                              | Type   | Status | Support Version |
 |-------------------------------------------------------------|--------|--------|-----------------|
+| [AmazonDynamoDB](connector-v2/sink/AmazonDynamoDB.md)       | Sink   | Beta   | 2.3.0           |
+| [AmazonDynamoDB](connector-v2/source/AmazonDynamoDB.md)     | Source | Beta   | 2.3.0           |
 | [Asset](connector-v2/sink/Assert.md)                        | Sink   | Beta   | 2.2.0-beta      |
-| [ClickHouse](connector-v2/source/Clickhouse.md)             | Source | Beta   | 2.2.0-beta      |
-| [ClickHouse](connector-v2/sink/Clickhouse.md)               | Sink   | Beta   | 2.2.0-beta      |
-| [ClickHouseFile](connector-v2/sink/ClickhouseFile.md)       | Sink   | Beta   | 2.2.0-beta      |
-| [Console](connector-v2/sink/Console.md)                     | Sink   | Beta   | 2.2.0-beta      |
+| [Cassandra](connector-v2/sink/Cassandra.md)                 | Sink   | GA     | 2.3.0           |
```
Comment (hailin0) [inline docs/en/Connector-v2-release-state.md]: Has anyone tested this connector?

Comment (CalvinKirs) [reply docs/en/Connector-v2-release-state.md]: I revert:)

File: docs/en/Connector-v2-release-state.md (hunk 1 above)
Comment (ashulin) [reply docs/en/Connector-v2-release-state.md:60]: Yes, it's GA status

**Review by hailin0 — APPROVED** [review]

**Review by TyrantLucifer — APPROVED** [review]

**Pull request merged**